package.json

```diff
@@ -119,7 +119,7 @@
 		"eslint-plugin-react": "^7.24.0",
 		"eslint-plugin-react-hooks": "^4.2.0",
 		"express": "^4.14.0",
-		"express-openapi-validator": "4.7.1",
+		"express-openapi-validator": "^4.13.2",
```

Comment:
I remember the uopdate for this extension caused an issue at some point. We need to check

Reply:
last time there was a problem with validation of the data type on students, in some fields where relation in open api yml was "anyOf". But this seems to be not the case anymore